taxi/views.py

```diff
@@ -36,6 +37,14 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(ManufacturerListView, self).get_context_data(**kwargs)
+        name = self.request.GET.get("name", "")
+        context["search_form"] = ManufacturerSearchForm(
+            initial={"username": name}
```

Comment:
The `ManufacturerListView` context data is incorrectly setting the initial value for the `username` field in the `ManufacturerSearchForm`. It should be `name` instead of `username` to match the form's field.